experimental/devices/mpu9250/mpu9250.go

```diff
@@ -0,0 +1,1936 @@
+}
+
+func (m *MPU9250) setOffset(regHi, regLo byte, offset uint16) error {
+	msb_offset := byte(offset >> 8)
+	lsb_offset := byte(offset & 0x00FF)
+	if err := m.transport.writeByte(regHi, msb_offset); err != nil {
+		return err
+	}
+	return m.transport.writeByte(regLo, lsb_offset)
+}
+
+// GetXGyroOffset Returns the offset of X gyroscope.
+func (m *MPU9250) GetXGyroOffset() (uint16, error) {
+	return m.getOffset(reg.MPU9250_XG_OFFSET_H, reg.MPU9250_XG_OFFSET_L)
+}
+
+// SetXGyroOffset Sets the X gyroscope offset.
+//
+//	offset offset value.
+func (m *MPU9250) SetXGyroOffset(offset uint16) error {
+	return m.setOffset(reg.MPU9250_XG_OFFSET_H, reg.MPU9250_XG_OFFSET_L, offset)
+}
+
+// GetYGyroOffset Returns the offset of y gyroscope.
+func (m *MPU9250) GetYGyroOffset() (uint16, error) {
+	return m.getOffset(reg.MPU9250_YG_OFFSET_H, reg.MPU9250_YG_OFFSET_L)
+}
+
+// SetZGyroOffset Sets the Z gyroscope offset.
+//
+//	offset offset value.
+func (m *MPU9250) SetZGyroOffset(offset uint16) error {
+	return m.setOffset(reg.MPU9250_ZG_OFFSET_H, reg.MPU9250_ZG_OFFSET_L, offset)
+}
+
+// GetZGyroOffset Returns the offset of Z gyroscope.
+func (m *MPU9250) GetZGyroOffset() (uint16, error) {
+	return m.getOffset(reg.MPU9250_ZG_OFFSET_H, reg.MPU9250_ZG_OFFSET_L)
+}
+
+// SetYGyroOffset Sets the Y gyroscope offset.
+//
+//	offset offset value.
+func (m *MPU9250) SetYGyroOffset(offset uint16) error {
+	return m.setOffset(reg.MPU9250_YG_OFFSET_H, reg.MPU9250_YG_OFFSET_L, offset)
+}
+
+// EnableAccelerometerAxis Accelerometer functions
+//
+//	axis MPU9250_DISABLE_XA_MASK, MPU9250_DISABLE_YA_MASK, MPU9250_DISABLE_ZA_MASK
+func (m *MPU9250) EnableAccelerometerAxis(axis byte) error {
+	return m.transport.writeMaskedReg(reg.MPU9250_PWR_MGMT_2, axis, 0)
+}
+
+// DisableAccelerometerAxis Accelerometer functions
+//
+//	axis MPU9250_DISABLE_XA_MASK, MPU9250_DISABLE_YA_MASK, MPU9250_DISABLE_ZA_MASK
+func (m *MPU9250) DisableAccelerometerAxis(axis byte) error {
+	return m.transport.writeMaskedReg(reg.MPU9250_PWR_MGMT_2, axis, axis)
+}
+
+// EnableAccelerometer enables the accelerometer.
+func (m *MPU9250) EnableAccelerometer() error {
+	return m.transport.writeMaskedReg(reg.MPU9250_PWR_MGMT_2, reg.MPU9250_DISABLE_XYZA_MASK, 0)
+}
+
+// GetAccelerometerTestData Get the self-test data values from the registers.
+func (m *MPU9250) GetAccelerometerTestData() (*AccelerometerData, error) {
+	x, err := m.transport.readByte(reg.MPU9250_SELF_TEST_X_ACCEL)
+	if err != nil {
+		return nil, err
+	}
+	y, err := m.transport.readByte(reg.MPU9250_SELF_TEST_Y_ACCEL)
+	if err != nil {
+		return nil, err
+	}
+	z, err := m.transport.readByte(reg.MPU9250_SELF_TEST_Z_ACCEL)
+	if err != nil {
+		return nil, err
+	}
+	return &AccelerometerData{X: int16(x), Y: int16(y), Z: int16(z)}, nil
+}
+
+func negateBool(src bool, err error) (bool, error) {
+	if err != nil {
+		return src, err
+	} else {
+		return !src, err
+	}
+}
+
+// AccelerometerXIsEnabled Get the X accelerometer status.
+func (m *
```

Comment:
This breaks `go vet -all -unsafeptr=false ./...` :(
https://gist.github.com/gohci-bot/dc3dc54e78a61be01f273a553e3719d2#file-cmd3-failed-in-4-487s
```
experimental\devices\mpu9250\mpu9250.go:1885: method WriteByte(address, value byte) error should have signature WriteByte(byte) error
```

Reply:
Sorry, I messed up with the branches, pushed the correct one. `go vet -all` passed just fine.